chain/network/src/types.rs

```diff
@@ -491,14 +491,17 @@ pub enum RegisterPeerResponse {
     Reject,
 }
 
+#[cfg(feature = "test_features")]
 #[cfg_attr(feature = "deepsize_feature", derive(DeepSizeOf))]
 #[derive(Clone, Debug)]
 pub struct GetPeerId {}
 
+#[cfg(feature = "test_features")]
 impl Message for GetPeerId {
     type Result = GetPeerIdResult;
 }
 
+#[cfg(feature = "test_features")]
 #[derive(MessageResponse, Debug)]
 #[cfg_attr(feature = "test_features", derive(serde::Serialize))]
```

Comment:
Given this structure will only be defined under `test_features`, we can flatten the `cfg_attr`:

```suggestion
#[cfg(feature = "test_features")]
#[derive(Debug, MessageResponse, serde::Serialize)]
```

P.S. Do we need `serde` here?

Reply:
@frol Yes, we use `GetPeerIdResult` in `jsonrpc` for testing purposes.